rmf_site_format/src/asset_source.rs

```diff
@@ -40,6 +41,7 @@ impl AssetSource {
             Self::Search(_) => "Search",
             Self::Bundled(_) => "Bundled",
             Self::Package(_) => "Package",
+            Self::OSMSlippyMap(_, _, _) => "Map",
```

Comment:
Consider changing this to something like OSM Tile

Reply:
Worth noting that the protocol is called "slippy tiles"

[fc2c182](https://github.com/open-rmf/rmf_site/pull/133/commits/fc2c182d5679b8cf20298741363d855b0c0b2e5d)